web/src/routes/System.jsx

```diff
@@ -347,7 +347,19 @@ export default function System() {
                     >
                       <div className="capitalize text-lg flex justify-between p-4">
                         <Link href={`/cameras/${camera}`}>{camera.replaceAll('_', ' ')}</Link>
-                        <Button onClick={(e) => onHandleFfprobe(camera, e)}>ffprobe</Button>
+                        <div className="flex">
+                          {(config.cameras[camera]['visit_url'] != "") && (
+                            <Link
```

Comment:
I think it would be better to make this a button that can have the aria-label explain that the UI link might not work, and also have the button like @hawkeye217 suggested

Reply:
Makes sense, I will change this.